build/yamls/antrea.yml

```diff
@@ -1467,12 +1468,17 @@ data:
 
     # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
     #tlsMinVersion:
+
+    # Enable namespace label mutator webhook to mutate/validate resources. For example, it
+    # enables the labels mutator which labels all Namespaces with the reserved
+    # "antrea.io/metadata.name = <namespaceName>" label.
+    #enableNamespaceLabelMutator: false
```

Comment:
a list would have made sense, but it makes it harder to manage defaults and when you enable some webhooks by default, you end up having to provide the ability to disable them. So if you think we will only have a couple of them at most, then individual config parameters make sense to me.

Reply:
agree